src/Libraries/RevitServices/Events/EventHandlerProxy.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Threading;
+using Autodesk.Revit.DB.Events;
+using Autodesk.Revit.UI.Events;
+using Dynamo.Models;
+
+namespace RevitServices.EventHandler
+{
+    /// <summary>
+    /// This is a event handler proxy class to serve as a proxy between the event publisher and
+    /// the event subscriber
+    /// </summary>
+    public class EventHandlerProxy
```

Comment:
Can we make `EventHandlerProxy` be a inner class of application? I don't think `EventHandlerProxy` is for public use, and probably should be sealed.